tests/test_proxy_functional.py

```diff
@@ -3,13 +3,52 @@
 import pathlib
 from unittest import mock
 
+import proxy
 import pytest
+import trustme
 from yarl import URL
 
 import aiohttp
 from aiohttp import web
 
 
+@pytest.fixture
+def trustme_certs(tmp_path):
+    ca = trustme.CA()
```

Comment:
Why didn't you reuse the trustme fixtures that already exist? Do these tests *need* an additional CA? Do you expect to have certificates for the proxy and the web server from different CA?

I think the simplest test could just rely on existing fixtures.

Reply:
I'm not familiar with the existing fixtures. I will look at replacing mine with those now.

Reply:
They are in conftest and are also listed @ `pytest --collect-only --fixtures`